File: Modul-06/SOAL 3/PRAK603-2310817210009-MuhammadIbnuSina.py
print("Input: ")
A, B = map(int, input().split())
if A != B:
    print('\nJumlah tidak sama')
    exit()

array1 = [int(j) for j in input().split()]
array2 = [int(j) for j in input().split()]

print("\nOutput: ")

for i in range(A):
    print(array1[i] * array2[i], end=" ")

File: Modul-06/SOAL 3/PRAK603-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int A, B;
    printf("Input: \n");
    scanf("%d %d", &A, &B);

    if (A != B) {
        printf("\nJumlah tidak sama\n");
    }

    int array1[A];
    int array2[A];

    for (int i = 0; i < A; i++) {
        scanf("%d", &array1[i]);
    }

    for (int i = 0; i < A; i++) {
        scanf("%d", &array2[i]);
    }

    printf("\nOutput: \n");

    for (int i = 0; i < A; i++) {
        printf("%d ", array1[i] * array2[i]);
    }

    return 0;
}
